src/windows/server_list.c

```diff
@@ -743,7 +743,12 @@ static void fetch_servers_callback(http_json_response* response)
 		SafeFree(newserver->version);
 		newserver->name = _strdup(json_string_value(name));
 		newserver->requiresPassword = json_boolean_value(requiresPassword);
-		newserver->description = _strdup(json_string_value(description));
+		if (description)
```

Comment:
Please use explicit comparisons (e.g. != NULL) when the expression is not a boolean.